composer.json

```diff
@@ -38,7 +38,8 @@
         "mccool/laravel-auto-presenter": "^4.3",
         "pragmarx/google2fa": "^0.7.1",
         "rcrowe/twigbridge": "^0.9.2",
-        "roumen/feed": "^2.10"
+        "roumen/feed": "^2.10",
+        "fedeisas/laravel-mail-css-inliner": "^1.5"
```

Comment:
We will need our own package since laravel-mail-css-inliner it's not compatible with 2.x `Error: Call to undefined method TijsVerkoyen\CssToInlineStyles\CssToInlineStyles::setUseInlineStylesBlock()`